x-pack/test/cases_api_integration/security_and_spaces/tests/trial/delete_sub_privilege.ts

```diff
@@ -0,0 +1,224 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { CASES_URL } from '@kbn/cases-plugin/common';
+import { FtrProviderContext } from '../../../common/ftr_provider_context';
+
+import { getPostCaseRequest, postCommentUserReq } from '../../../common/lib/mock';
+import {
+  deleteCasesByESQuery,
+  deleteCasesUserActions,
+  deleteComments,
+  createCase,
+  deleteCases,
+  createComment,
+  getCase,
+  superUserSpace1Auth,
+  findCases,
+  deleteComment,
+  getSpaceUrlPrefix,
+  updateComment,
+} from '../../../common/lib/utils';
+import { superUser, secOnlyDelete } from '../../../common/lib/authentication/users';
+
+// eslint-disable-next-line import/no-default-export
+export default ({ getService }: FtrProviderContext): void => {
+  const supertestWithoutAuth = getService('supertestWithoutAuth');
+  const es = getService('es');
+
+  describe('user with deletion sub privilege', () => {
+    afterEach(async () => {
+      await deleteCasesByESQuery(es);
+      await deleteComments(es);
+      await deleteCasesUserActions(es);
+    });
+
+    describe('successful operations', () => {
+      it(`should delete a case`, async () => {
+        const postedCase = await createCase(
+          supertestWithoutAuth,
+          getPostCaseRequest({ owner: 'securitySolutionFixture' }),
+          200,
+          {
+            user: superUser,
+            space: 'space1',
+          }
+        );
+
+        await deleteCases({
+          supertest: supertestWithoutAuth,
+          caseIDs: [postedCase.id],
+          expectedHttpCode: 204,
+          auth: { user: secOnlyDelete, space: 'space1' },
+        });
+      });
+
+      it(`should delete a comment from the appropriate owner`, async () => {
```

Comment:
Could you please add a test where the `secOnlyDelete` can delete a case with comments?

Reply:
Good ideas, done.